_layouts/default.html

```diff
@@ -7,173 +7,219 @@
-            </header>
-
-            <div class="documentation">
-                <div class="row">
-                    <div class="col-xxl-2 col-md-3">
-                        <div class="doc-sidebar">
-                            {% if site.search.enabled == true %}
-                            <form class="doc-search" action="{{ site.search.url | relative_url }}">
-                                <input type="text" class="form-control" name="q" id="tipue_search_input" placeholder="Search..." autocomplete="off" pattern=".{3,}" title="At least 3 characters">
-                            </form>
-                            {% endif %}
-
-                            {% if page.sidebar.navigation != nil and page.sidebar.navigation != empty %}
-                            <nav class="documentation-nav">
-                                {% for group in page.sidebar.navigation %}
-                                {% comment %}
-                                TODO: Make "class='active'" conditional on whether one of the group's nav items are active or not
-                                {% endcomment %}
-                                <div class="nav-group active">
-                                        <div class="nav-heading">
-                                            <i class="material-icons">keyboard_arrow_right</i>
-                                            <span>{{ group.title }}</span>
-                                        </div>
-
-                                        {% if group.items != nil and group.items != empty %}
-                                        <ul>
-                                        {% for nav in group.items %}
-                                            {% assign nav_url = nav.url | relative_url %}
-                                            <li><a {% if page_url == nav_url %} aria-current="page" class="active" {% endif %} href="{{ nav_url }}"><span>{{ nav.title }}</span></a></li>
-                                        {% endfor %}
-                                        </ul>
-                                        {% endif %}
-                                    </div>
+                </div>
+            </nav>
+        </header>
+
+        <div class="documentation">
+            <div class="row">
+                <div class="col-xxl-2 col-md-3">
+                    <div class="doc-sidebar">
+                        {% if site.search.enabled == true %}
+                        <form class="doc-search" action="{{ site.search.url | relative_url }}">
+                            <input type="text" class="form-control" name="q" id="tipue_search_input"
+                                placeholder="Search..." autocomplete="off" pattern=".{3,}"
+                                title="At least 3 characters">
+                        </form>
+                        {% endif %}
+                        {% if page.sidebar.navigation != nil and page.sidebar.navigation != empty %}
```

Comment:
Didn't we change this to not check for `empty`? Or does it work regardless?

Reply:
The check does not make a difference, just tested it. I'll check in without.